features/gift_exchanges/challenge_giftexchange.feature

```diff
@@ -382,15 +382,23 @@ Feature: Gift Exchange Challenge
       And I should see "Offers by myname1"
       But I should not see the image "alt" text "email myname1"
 
-  Scenario: User fulfills their assignment and it shows on their assigments page as fulfilled
+  Scenario: User fulfills their assignment and it shows on their assignments page as fulfilled
 
     Given everyone has their assignments for "Awesome Gift Exchange"
     When I am logged in as "myname1"
-      And I fulfill my assignment
-    When I go to myname1's user page
+      And I go to myname1's user page
       And I follow "Assignments"
```

Comment:
We can tighten these tests up a bit by using `I go to the assignments page for "user"` instead of going to the user page and following the Assignments link all the time.